integrations/zendesk/src/actions/hitl.ts

```diff
@@ -0,0 +1,50 @@
+import { getZendeskClient } from '../client'
+import * as bp from '.botpress'
+
+export const startHitl: bp.IntegrationProps['actions']['startHitl'] = async ({ ctx, input, client }) => {
+  const zendeskClient = getZendeskClient(ctx.configuration)
+
+  const { user } = await client.getUser({
+    id: input.userId,
+  })
+
+  const ticket = await zendeskClient.createTicket(input.title, input.description ?? '...', {
+    name: user.name ?? '',
+    email: user.tags.email ?? '',
+  })
+
+  const { conversation } = await client.getOrCreateConversation({
+    channel: 'hitl',
+    tags: {
+      id: `${ticket.id}`,
+    },
+  })
+
+  return {
+    conversationId: conversation.id,
+  }
+}
+
+export const stopHitl: bp.IntegrationProps['actions']['stopHitl'] = async ({ ctx, input, client }) => {
```

Comment:
HITLINT change number 3) Allows bot to stop conversation with agent, and kick the user back to the workflow.